src/app/pages/send/sent-summary/sent-summary.layout.tsx

```diff
@@ -0,0 +1,69 @@
+import { FiCheck, FiCopy, FiExternalLink } from 'react-icons/fi';
+
+import { Stack } from '@stacks/ui';
+
+import { FormAddressDisplayer } from '@app/components/address-displayer/form-address-displayer';
+import {
+  InfoCard,
+  InfoCardAssetValue,
+  InfoCardBtn,
+  InfoCardFooter,
+  InfoCardRow,
+  InfoCardSeparator,
+} from '@app/components/info-card/info-card';
+
+interface SentSummaryDetailsProps {
+  arrivesIn: string;
+  fee: string;
+  onClickCopy(): void;
+  onClickLink(): void;
+  recipient: string;
+  sendingValue: string;
+  symbol: string;
+  totalSpend: string;
+  txValue: number;
+  txFiatValue: string;
+  txFiatValueSymbol: string;
+}
+export function SentSummaryLayout({
+  arrivesIn,
+  fee,
+  onClickCopy,
+  onClickLink,
+  recipient,
+  sendingValue,
+  symbol,
+  totalSpend,
+  txValue,
+  txFiatValue,
+  txFiatValueSymbol,
+}: SentSummaryDetailsProps) {
+  return (
+    <InfoCard>
+      <InfoCardAssetValue
+        value={txValue}
+        fiatValue={txFiatValue}
+        fiatSymbol={txFiatValueSymbol}
+        symbol={symbol}
+        icon={FiCheck}
+        my="loose"
+        px="loose"
+      />
+      <Stack width="100%" px="extra-loose" pb="extra-loose">
+        <InfoCardRow title="To" value={<FormAddressDisplayer address={recipient} />} />
+        <InfoCardSeparator />
+        <InfoCardRow title="Total spend" value={totalSpend} />
+
+        <InfoCardRow title="Sending" value={sendingValue} />
+        <InfoCardRow title="Fee" value={fee} />
+        <InfoCardRow title="Estimated confirmation time" value={arrivesIn} />
+      </Stack>
+      <InfoCardFooter>
+        <Stack spacing="base" isInline width="100%">
+          <InfoCardBtn onClick={onClickLink} icon={FiExternalLink} label="View Details" />
+          <InfoCardBtn onClick={onClickCopy} icon={FiCopy} label="Copy ID" />
+        </Stack>
+      </InfoCardFooter>
+    </InfoCard>
+  );
+}
```

Comment:
I'm torn on this abstraction. There's very little layout here. Other than the 2x stacks, everything else is content.

To avoid repeating the content, we create the abstraction of this component, which comes at the cost of 1) creating a load of props that need passing down, 2) introducing inflexibility that if we wanted to change one, and not the other, we can't. 

Not suggesting this needs changing, it comes with some benefits, but think it's worth weighing up the pros/cons.

Reply:
I just don't think I named this well, so will change, but I need to reuse it and seems odd to have the same code in three places now rather than share it. `SendSummaryDetails`?

Reply:
Actually, ended up agreeing here while refactoring and putting back. Not sure tbh why this needs to be two components witth the send form bc only the symbol changes, but not scoped to this work. I'll address the summary for transfer separately.